lib/foreman_tasks/engine.rb

```diff
@@ -39,6 +39,12 @@ class Engine < ::Rails::Engine
       Foreman::Gettext::Support.add_text_domain locale_domain, locale_dir
     end
 
+    initializer 'foreman_tasks.rails_executor_middleware' do
+      Actions::Base.class_eval do
+        middleware.use ::Actions::Middleware::RailsExecutorWrap if SETTINGS[:rails] != '4.2'
```

Comment:
Any reason to not put it directly to https://github.com/theforeman/foreman-tasks/blob/212ec9a066187a9e3bfebeb9a20283596ba585d0/app/lib/actions/base.rb#L2 ?

Reply:
For some reason i thought that class was in dynflow, i'm not really sure why i thought that